packages/venia-concept/package.json

```diff
@@ -45,7 +46,8 @@
     "apollo-link-context": "^1.0.14",
     "apollo-link-http": "^1.5.11",
     "braintree-web-drop-in": "^1.16.0",
-    "graphql": "^14.1.1",
+    "graphql-cli": "^3.0.11",
+    "graphql-cli-validate-magento-pwa-queries": "^1.0.0",
```

Comment:
This is really a mouthful. 😆

Reply:
Lol yeah, it was much better as `validate-queries` but then we made it magento / pwa specific 😝